neuralforecast/models/timemixer.py

```diff
@@ -200,13 +201,13 @@ def __init__(self, seq_len, down_sampling_window, down_sampling_layers):
             [
                 nn.Sequential(
                     torch.nn.Linear(
-                        seq_len // (down_sampling_window ** (i + 1)),
-                        seq_len // (down_sampling_window**i),
+                        math.ceil(seq_len / (down_sampling_window ** (i + 1))),
+                        math.ceil(seq_len / (down_sampling_window**i)),
                     ),
                     nn.GELU(),
                     torch.nn.Linear(
-                        seq_len // (down_sampling_window**i),
-                        seq_len // (down_sampling_window**i),
+                        math.ceil(seq_len // (down_sampling_window**i)),
```

Comment:
Here `math.ceil` isn't necessary. Above it is. Should we make it consistent? I.e. either use `//` in both cases or use `/` with `math.ceil`?